docs/versioned_docs/version-2.50.0-LTS/contributing-guide/setup/macos.md

```diff
@@ -40,25 +40,32 @@ To set up and run ToolJet on macOS for development, begin by opening your termin
 
     :::info 
     Required only if ToolJet Database is being used. 
-    :::       
+    :::
+
+    To install a specific version of PostgRest using Homebrew:
+
+    ```bash
+    brew install postgrest@<version>
+    ```
+    To find out which versions are available, you can search with:
 
     ```bash
-    brew install postgrest
+    brew search postgrest
```

Comment:
You can use `brew services list` instead. This command will help you get a list of running services along with additional details.

Reply:
Yes, I have added this command in the latest commit to let user understand it more clearly